programmers/피보나치-수.js

```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
+const input = parseInt(fs.readFileSync('/dev/stdin').toString());
+
+function fibonacci(n) {
+    if (n === 0 || n === 1) return n;
```

Comment:
```suggestion
    if (n <= 1) return n;
```
당연히 알겠지만 이방법도 있습니다  !